packages-exp/messaging-exp/src/util/errors.ts

```diff
@@ -50,6 +51,8 @@ export const ERROR_MAP: ErrorMap<ErrorCode> = {
     'The notification permission was not granted and blocked instead.',
   [ErrorCode.UNSUPPORTED_BROWSER]:
     "This browser doesn't support the API's required to use the firebase SDK.",
+  [ErrorCode.INDEXED_DB_UNSUPPORTED]:
+    "This browser context doesn't support the idb.open (ex. Safari iFrame, Firefox Private Browsing, etc)",
```

Comment:
Suggest "doesn't support indexedDB.open()" (no "the") or "doesn't support the indexedDB.open() method"